test/cmdlineTests/standard_model_checker_solvers_smtlib2/output.json

```diff
@@ -30,8 +30,7 @@
 (check-sat)
 (get-value (|EVALEXPR_0| ))
 ",
-            "0xa991e3c158410479bc0a0540fb60ce7010aec315a5b0010d254f12f3d1f0a4e5": "(set-logic HORN)
-
+            "0xe8a37a28c987dde3c33e9b37ef24bde452691ace980b53b1aff16e2fa2e97fba": "(set-logic HORN)
```

Comment:
@leonardoalt  what is this hash?

Reply:
It has changed and  I don't know why.

Reply:
This is the hash of the query string. It changes whenever the query itself changes.

Reply:
I'm wondering if some `\n` changed